test-requirements.txt

```diff
@@ -5,3 +5,4 @@ nose
 nosexcover
 openstack.nose_plugin
 nosehtmloutput
+mock>=1.0
```

Comment:
Doesn't this conflict with the `mock` requirement set in `tox.ini` with Python-version-specific bounds?